events/2019/2-hardware.md

```diff
@@ -4,11 +4,19 @@ date: 2019-08-17
 actions:
   -
     title: Sign up for Hardware
-    url: http://www.meetup.com/JSOxford/events/231241496/
+    url: https://www.meetup.com/Summer-Of-Hacks-Oxford/events/258795349/
 ---
 
-This is a day of hacking on hardware. It's for anyone who is interested in learning about hardware and there is no minimum entry requirememts, other than an equisitive mind.
+This is a day of hacking on hardware. It's for anyone who is interested in
```

Comment:
There were a handful of typos in this event description which I opted to fix inline with this PR.

Reply:
the students has become the pypobot!